vkdgen.py

```diff
@@ -256,8 +256,9 @@ def genStruct(self, typeinfo, name):
 		print("}", file=self.typesFile)
 	
 	def genGroup(self, groupinfo, name):
-		super().genGroup(groupinfo, name)
-		print("enum %s {" % name, file=self.typesFile)
+		enumName = name

```

Comment:
In case you're wondering, I need to save the original enum name here so that I can use it in line 306.